Rewatch.lua

```diff
@@ -1632,7 +1822,7 @@ rewatch_events:SetScript("OnUpdate", function()
 				x, y = UnitPowerMax(v["Player"]), UnitPower(v["Player"]);
 				v["ManaBar"]:SetMinMaxValues(0, x); v["ManaBar"]:SetValue(y);
 				-- fade when out of range
-				if(IsSpellInRange(rewatch_loc["healingtouch"], v["Player"]) == 1) then
+				if(IsSpellInRange(rewatch_loc["regrowth"], v["Player"]) == 1 or IsSpellInRange(rewatch_loc["healingwave"], v["Player"]) == 1) then
```

Comment:
We have some logic above from earlier which also checks for a non-cd-spell - best to store it locally and use here, as well.